src/main/java/address/model/AddPersonCommand.java

```diff
@@ -82,7 +85,10 @@ protected void after() {
     protected State simulateResult() {
         assert input != null;
         // create VP and add to model
-        viewableToAdd = PlatformExecUtil.callAndWait(() -> model.addViewablePersonWithoutBacking(input), null);
+        PlatformExecUtil.runAndWait(() -> {
+            viewableToAdd = model.addViewablePersonWithoutBacking(input);
+            viewableToAdd.setChangeInProgress(ADDING);
```

Comment:
No, but the status setting makes more sense existing in the command than the model code